apps/nextjs/src/app/[locale]/boards/(content)/_header-actions.tsx

```diff
@@ -151,6 +156,27 @@ const EditModeMenu = () => {
   );
 };
 
+const SelectBoardsMenu = () => {
+  const { data: boards = [] } = clientApi.board.getAllBoards.useQuery();
+
+  return (
+    <Menu position="bottom-end" withArrow>
+      <Menu.Target>
+        <HeaderButton w="auto" px={4}>
+          <IconReplace stroke={1.5} />
+        </HeaderButton>
+      </Menu.Target>
+      <Menu.Dropdown style={{ transform: "translate(-3px, 0)" }}>
```

Comment:
What exactly is the goal of this translate? For me moving it to the center of the button would be `translate(-7px, 0)`